examples/passport-login/src/index.ts

```diff
@@ -83,6 +85,9 @@ export async function main() {
   } else {
     oauth2Providers = require('../oauth2-providers');
   }
-  const server: ExpressServer = await startApplication(oauth2Providers);
+  const server: ExpressServer = await startApplication(
+    oauth2Providers,
+    '../data/db.json',
```

Comment:
I think it still creates `examples/passport-login/data/db.json`. What about creating it under `examples/passport-login/dist/data/db.json` or `examples/passport-login/.sandbox/data/db.json`?

Reply:
The other option is to add `data/db.json` to `.gitignore`.

Reply:
it wouldn't use the file for the tests, because the test script calls startApplication() directly without the second parameter. Here, we have just moved the hardcoded memory file parameter to the main() method that is called only when doing an `npm start` from example root.

Reply:
@raymondfeng I have added data/db.json to .gitignore

Reply:
@raymondfeng the acceptance test will not change the db.json file anymore, since it doesnt pass in the json file path.
https://github.com/strongloop/loopback-next/blob/05370bcc4cc7707ffd018cd6fbfc925935b05a8d/examples/passport-login/src/__tests__/acceptance/passport-login.acceptance.ts#L43-L47